--- a/Tests/mocks/thunder/ServiceMock.h
+++ b/Tests/mocks/thunder/ServiceMock.h
@@ -28,7 +28,7 @@ class ServiceMock : public PluginHost::IShell {
 public:
     virtual ~ServiceMock() = default;
 
-    MOCK_METHOD(uint32_t, AddRef, (), (const, override));
+    MOCK_METHOD(void, AddRef, (), (const, override));
     MOCK_METHOD(uint32_t, Release, (), (const, override));
     MOCK_METHOD(void*, QueryInterface, (const uint32_t), (override));
 
--- a/Tests/mocks/USBDeviceMock.h
+++ b/Tests/mocks/USBDeviceMock.h
@@ -36,7 +36,7 @@ class USBDeviceMock : public Exchange::IUSBDevice {
 public:
     virtual ~USBDeviceMock() = default;
 
-    MOCK_METHOD(uint32_t, AddRef, (), (const, override));
+    MOCK_METHOD(void, AddRef, (), (const, override));
     MOCK_METHOD(uint32_t, Release, (), (const, override));
     MOCK_METHOD(void*, QueryInterface, (const uint32_t), (override));
 
